drake/util/drakeAppUtil.h

```diff
@@ -0,0 +1,40 @@
+#ifndef DRAKE_DRAKEAPPUTIL_H
+#define DRAKE_DRAKEAPPUTIL_H
+
+#include <string>
+#include <algorithm>
+
+/*** getCommandLineOption
+ * @brief Provides a platform-independent way to parse command line options (getopt is not available on msvc)
+ *
+ * Example usage:
+ * char * filename = getCommandLineOption(argv, argv + argc, "-f");
+ * if (filename) { ... }
+ *
+ * See also commandLineOptionExists
+ */
+char* getCommandLineOption(char ** begin, char ** end, const std::string & option)
```

Comment:
i searched for one, but most don't support windows.  the only notable exception was boost.  this was the recommended cross-platform solution.